### PR TITLE
Ignore non-pr payloads

### PR DESCRIPTION
**Pull request by gylaz**
Don't schedule jobs for payloads that are not for a pull request
(e.g. commit push). No reason to waste the backround job round trip just to
do nothing.

### CODE REVIEW

File: spec/models/payload_spec.rb [hunk 1]
```diff
@@ -52,6 +53,28 @@
...
+      it "returns false" do
+        push_event = File.read("spec/support/fixtures/push_event.json")
+        payload = Payload.new(push_event)
+
+        expect(payload).not_to be_pull_request
+      end
+    end
+
+    context "when payload for pull request" do
+      it "returns false" do
```
Comment (thorncp) [inline spec/models/payload_spec.rb]: `false` -> `true`

**Comment by thorncp** [comment]
Does this mean hound will no longer run when new commits are added to an open pull request?

**Comment by croaky** [comment]
So this checks that it is a pull request, but we have other code that checks the `synchronize` event later, right? https://developer.github.com/v3/activity/events/types/#pullrequestevent

**Comment by gylaz** [comment]
@thorncp No, that is considered a pull request event with an "synchronize" action. This filters our the push events to a branch, that's not been PR'd.

@croaky Yep. [`BuildRunner#run`](https://github.com/thoughtbot/hound/blob/master/app/services/build_runner.rb#L5) has a check for `opened` and `synchronize` type actions.

**Comment by salbertson** [comment]
:+1:

**Pull request merged**